README.md

```diff
@@ -39,55 +45,18 @@ versions.
 
 </details>
 
-Anaconda:
-
-```
-conda install torchvision -c pytorch
-```
-
-pip:
-
-```
-pip install torchvision
-```
+## Image Backends
```

Comment:
I'm not sure this section is relevant these days. I'm tempted to remove it. @pmeier @vfdev-5 any thought?

Reply:
OK to me

Reply:
I guess I'll just keep it for now. I wouldn't even know what to do with the "video backend" section either...